### ISSUE TITLE
Oregon base station compatibility

### ISSUE THREAD

**Issue by karioja** [deberman/OsV3_wind_sensor#2]
Did you ever manage to verify if this works with a genuine Oregon base station? I've implemented a THGR810 temp sensor (using this wind sensor as a basic) and I'm able to receive the data using my own custom Arduino receiver as well as using a RTL-SDR dongle (using rtl_433). However my WMR86 base station fails to receive the data. I've captured the waveforms (my own transmitter and a real THGR810) with rtl_433 and visualized them using http://triq.net/iqs. They seem identical. Any hints?

**Comment by deberman** [comment]
No, I have a Telldus net stick that I tested against. It was quite some time i did this but I remember when reading the source code for the stick there where comments like ”what does this bit do? Ignore until we know more” That might be something to look into and explain why my stick accepts the message but not your base station. 
Link to telldus source:
http://developer.telldus.com/browser
